platform/settings.hpp

```diff
@@ -33,22 +31,22 @@ class StringStorage : public platform::StringStorageBase
 /// Retrieve setting
 /// @return false if setting is absent
 template <class Value>
-[[nodiscard]] bool Get(std::string const & key, Value & outValue)
+[[nodiscard]] bool Get(std::string_view key, Value & outValue)
```

Comment:
nit: std::optional is a good candidate for this case.

Reply:
Nah, std::string_view then or std::string const *

Reply:
- ```optional<string>``` is a strange pattern itself (we don't need an empty setting)
- avoid useless copy